tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -3320,6 +3320,7 @@ declare namespace ts {
          */
         resolveTypeReferenceDirectives?(typeReferenceDirectiveNames: string[] | readonly FileReference[], containingFile: string, redirectedReference: ResolvedProjectReference | undefined, options: CompilerOptions, containingFileMode?: SourceFile["impliedNodeFormat"] | undefined): (ResolvedTypeReferenceDirective | undefined)[];
         getEnvironmentVariable?(name: string): string | undefined;
+        /** If provided along with custom resolveModuleNames or resolveTypeReferenceDirectives, used to determine if unchanged file path needs to re-resolve modules/type reference directives */
```

Comment:
Does that mean anything that referenced this thing needs to re-resolve, or that everything that this path references needs to be re-resolved?

Reply:
When returned true for party, the module names  in that file will be re- resolved

Reply:
Should it have been `hasInvalidatedResolution`**`s`**?

Reply:
Sure .. I can only submit Pr for that tomorrow though

Reply:
No worries, I got you covered #50912